### PR TITLE
Added enable_implicit_transaction_for_batch_statements to v22.2 release notes

### PR DESCRIPTION
**Pull request by rafiss**
References https://github.com/cockroachdb/cockroach/pull/76834
See also DOC-6417

### CODE REVIEW

File: _includes/v22.2/backward-incompatible/alpha.1.md [hunk 1]
```diff
@@ -11,4 +11,5 @@
 - Removed the ability to cast `int`, `int2`, and `int8` to a `0` length `BIT` or `VARBIT`. [#81266][#81266]
 - Removed the deprecated `GRANT` privilege. [#81310][#81310]
 - Removed the `ttl_automatic_column` storage parameter. The `crdb_internal_expiration` column is created when `ttl_expire_after` is set and removed when `ttl_expire_after` is reset. [#83134][#83134]
-- Removed the byte string parameter in the `crdb_internal.schedule_sql_stats_compaction` function. [#82560][#82560]
\ No newline at end of file
+- Removed the byte string parameter in the `crdb_internal.schedule_sql_stats_compaction` function. [#82560][#82560]
+- Changed the default value of the `enable_implicit_transaction_for_batch_statements` to `true`. This means that a [batch of statements](../v22.2/transactions.html#batched-statements) sent in one string separated by semicolons is treated as an implicit transaction. [#76834](#76834)
```
Comment (nickvigilante) [inline _includes/v22.2/backward-incompatible/alpha.1.md:15]: At the bottom of the following two files, can you add the following line?

```
[#76834]: https://github.com/cockroachdb/cockroach/pull/76834
```

- _includes/releases/v22.2/v22.2.0-alpha.1.md
- _includes/releases/v22.2/v22.2.0.md

**Review by nickvigilante — CHANGES_REQUESTED** [review]
Two changes and this should be good.

File: _includes/v22.2/backward-incompatible/alpha.1.md (hunk 1 above)
Comment (rafiss) [reply _includes/v22.2/backward-incompatible/alpha.1.md:15]: i didn't see a place to put it in the _includes/releases/v22.2/v22.2.0.md file

Comment (nickvigilante) [inline _includes/v22.2/backward-incompatible/alpha.1.md:15]:
```suggestion
- Changed the default value of the `enable_implicit_transaction_for_batch_statements` to `true`. This means that a [batch of statements](../v22.2/transactions.html#batched-statements) sent in one string separated by semicolons is treated as an implicit transaction. [#76834][#76834]
```

**Review by nickvigilante — CHANGES_REQUESTED** [review]
Another change.

File: _includes/v22.2/backward-incompatible/alpha.1.md (hunk 1 above)
Comment (rafiss) [reply _includes/v22.2/backward-incompatible/alpha.1.md]: done, nice catch

**Review by rmloveland — APPROVED** [review]
LGTM, thanks @rafiss !

**Review by nickvigilante — APPROVED** [review]

**Pull request merged**